app/src/full/java/io/homeassistant/companion/android/sensors/GeocodeSensorManager.kt

```diff
@@ -92,7 +92,7 @@ class GeocodeSensorManager : SensorManager {
                 )
             }.orEmpty()
 
-            val prettyAddress = address?.getAddressLine(0) ?: context.getString(R.string.state_unknown)
+            val prettyAddress = address?.getAddressLine(0) ?: context.getString(R.string.unknown_address)
```

Comment:
Are we planning on skipping this string `context.getString(R.string.unknown_address)` in translations? Do we need to mark the string a special way?  Because right now this will send the translated string of `Unknown` to HA which is undesired when the address returns as null

Reply:
Ah sorry. Now i get it what you mean. Now it's ok i think?

Reply:
Yup looks like its ok now